Multitouch Support/Native/VoodooI2CNativeEngine.cpp

```diff
@@ -22,6 +22,7 @@ MultitouchReturn VoodooI2CNativeEngine::handleInterruptReport(VoodooI2CMultitouc
     for (int i = 0; i < event.contact_count; i++) {
         VoodooI2CDigitiserTransducer* transducer = (VoodooI2CDigitiserTransducer*) event.transducers->getObject(i);
         VoodooInputTransducer* inputTransducer = &message.transducers[i];
+        memset(inputTransducer, 0, sizeof(VoodooInputTransducer));
```

Comment:
Well I mean clean up the `message.transducers` before the loop. 😂
like this:
```diff
MultitouchReturn VoodooI2CNativeEngine::handleInterruptReport(VoodooI2CMultitouchEvent event, AbsoluteTime timestamp) {
    if (!voodooInputInstance) {
        return MultitouchReturnContinue;
    }

+   message = {};
    message.timestamp = timestamp;
    message.contact_count = event.contact_count;
    
    for (int i = 0; i < event.contact_count; i++) {
```
or
```diff
MultitouchReturn VoodooI2CNativeEngine::handleInterruptReport(VoodooI2CMultitouchEvent event, AbsoluteTime timestamp) {
    if (!voodooInputInstance) {
        return MultitouchReturnContinue;
    }

    message.timestamp = timestamp;
    message.contact_count = event.contact_count;
+   memset(message.transducers, 0, VOODOO_INPUT_MAX_TRANSDUCERS * sizeof(VoodooInputTransducer));
    
    for (int i = 0; i < event.contact_count; i++) {
```